lib/analyzer/applications/node-modules-utils.ts

```diff
@@ -32,6 +32,10 @@ async function createTempProjectDir(
 const manifestName: string = "package.json";
 
 async function fileExists(path: string): Promise<boolean> {
+  if (!path) {
```

Comment:
what is this checking?

Reply:
Oh, if the string is "".

Reply:
how come line 39-41 don't handle this edge case?

Reply:
it does, removing the unnecessary check. thanks!